packages/perseus/src/validation.types.ts

```diff
@@ -39,10 +38,14 @@ export type EmptyUserInput = Empty;
 // TODO (LEMS-2396): remove validation logic from widgets that don't validate
 export type NullUserInput = null;
 
-export type PerseusCategorizerRubric = PerseusCategorizerWidgetOptions;
+export type PerseusCategorizerRubric = {
+    // The correct answers where index relates to the items and value relates
+    // to the category.  e.g. [0, 1, 0, 1, 2]
+    values: ReadonlyArray<number>;
+};
 
 export type PerseusCategorizerUserInput = {
-    values: ReadonlyArray<number>;
+    values: PerseusCategorizerRubric["values"];
```

Comment:
This could instead just point directly to the Rubric type since the UserInput type is the same:
```
export type PerseusCategorizerUserInput = PerseusCategorizerRubric
```
Going with still showing the shape of the UserInput object, but could switch if anyone feels strongly about it.

Reply:
This makes sense to me 🤷